src/features/events/components/EventSelectionCheckBox.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Checkbox } from '@mui/material';
+import { useSelector, useStore } from 'react-redux';
+
+import { eventsSelected } from '../store';
+import { RootState } from 'core/store';
+import { ZetkinEvent } from 'utils/types/zetkin';
+
+interface EventSelectionCheckBoxProps {
+  eventList: ZetkinEvent[];
```

Comment:
I don't see `eventList` being used as a naming convention for naming simple arrays anywhere else, so I would just call this `events`.